### ISSUE TITLE
Switch using continue inside a loop

### ISSUE THREAD

**Issue by golddragon007** [kalessil/phpinspectionsea#1652]
Description (screenshot):
--
It might be interesting to add a warning for the following: when you have a switch case inside a loop and you want to continue with the next iteration from the switch case, you need to use `continue 2;` instead just `continue;` because inside the switch case the `continue;` means the same as the `break;`. Normally PHP warns about this, well if it's enabled the warnings to be displayed.

Example
--
Buggy (do again something will run after the continue was executed):
```php
foreach ($somthings as $key => $something) {
  // do something.
  switch ($key) {
    case 'a':
      // do something.
      if ($somevariable === null) {
        continue; // this will work in a switch case as a break;
      }
      // finally finishing some other stuff.
      break;
  }
  // do again something.
}
```
Correct (do again something will NOT run after the continue was executed, a new iteration will start immediatelly):
```php
foreach ($somthings as $key => $something) {
  // do something.
  switch ($key) {
    case 'a':
      // do something.
      if ($somevariable === null) {
        continue 2;
      }
      // finally finishing some other stuff.
      break;
  }
  // do again something.
}
```

**Comment by ea-inspections-team** [comment]
Hey @golddragon007, the inspection you are requesting is already available. It's cam be found in inspections setting under `PHP -> Php Inspections (EA *) -> Probable bugs -> Continue misbehaviour in switch`.
